app/models/school.rb

```diff
@@ -1,8 +1,4 @@
 class School < ActiveRecord::Base
-
-  attr_accessible :name
-
-  belongs_to :campus
```

Comment:
Hmm I'm a bit concerned we're removing this association. Are we 100% sure the association is not being called anywhere? In addition, why was campus initially added? What value did it provide besides for that fact of illustrating some schools that are located within the same area or build.

Reply:
Yes, I think it's safe to remove campus_id because of Courtney's comments: https://jira.nypl.org/browse/MLN-143

When you get a chance, let me show you in-person what I found when I searched the source code for dependencies on campus_id.